java/src/main/java/com/powsybl/python/network/NetworkAreaDiagramUtil.java

```diff
@@ -183,5 +183,11 @@ private void visitTerminal(Terminal terminal) {
             }
 
         }
+
+        public void visitDanglingLine(DanglingLine danglingLine) {
```

Comment:
it is not used anywhere ?

Reply:
It works the same way as visitHvdcConverterStation for example.